llvm/lib/Transforms/IPO/DynUntrustedAllocPost.cpp

```diff
@@ -50,6 +50,7 @@
 
 #define DEBUG_TYPE "dyn-untrusted"
 #define MPK_STATS
+#define MPK_PATCH_INST_DEBUG
```

Comment:
Can we get some comments on these? DEBUG_TYPE is an LLVM thing, so you can ignore, since it is fairly standard.

Reply:
Added comment to `MPK_STATS` and removed `MPK_PATCH_INST_DEBUG` in favor of a flag to activate printing of modified allocation sites.